vr-wordle/src/WordleLogic.js

```diff
@@ -0,0 +1,54 @@
+import { answerWords } from "./word-lists/answer-words.js";
+import { allowedWords } from "./word-lists/allowed-words.js";
+
+function getRandomAnswerWord(answerWords) {
+  let answer = answerWords[Math.floor(Math.random() * answerWords.length)];
+  return answer;
+}
+
+function compare(guess, allowedWords, answer) {
+  let result = [];
+  if (allowedWords.includes(guess)) {
+    for (let i = 0; i < guess.length; i++) {
+      if (guess[i] === answer[i]) {
+        result.push("🟩");
+      } else if (answer.indexOf(guess[i]) !== -1) {
+        result.push("🟨");
+      } else {
+        result.push("⬜️");
+      }
+    }
```

Comment:
These emojis might cause issues in implementation, we'll cross that bridge later though